README.md

```diff
@@ -48,6 +48,7 @@ and running tests have been implemented.
 | Protectli    | VP4650               | Dasharo                  |  `protectli-vp4650`                    |
 | Protectli    | VP4670               | Dasharo                  |  `protectli-vp4670`                    |
 | Raptor-CS    | TalosII              | Dasharo                  |  `raptor-cs_talos2`                    |
+| N/A          | RaspberryPi 3B       | Yocto                    |  `rpi-3b`                              |
```

Comment:
The vendor can simply by Raspberry Pi, not N/A

Reply:
@macpijan i added the vendor info in [this commit](https://github.com/Dasharo/open-source-firmware-validation/pull/22/commits/fe41929bdd7591ca142da1a439ed7e332afaf33a)